src/sql/base/browser/ui/table/plugins/buttonColumn.plugin.ts

```diff
@@ -57,8 +59,10 @@ export class ButtonColumn<T extends Slick.SlickData> implements Slick.Plugin<T>
 	private handleActiveCellChanged(args: Slick.OnActiveCellChangedEventArgs<T>): void {
 		if (this.isCurrentColumn(args.cell)) {
 			const cellElement = this._grid.getActiveCellNode();
-			const button = cellElement.children[0] as HTMLButtonElement;
-			button.focus();
+			if (cellElement && cellElement.children) {
```

Comment:
when will the children collection be undefined or empty?

Reply:
do I need to do that? haven't seen the same thing for radio button 

---
In reply to: [457876752](https://github.com/microsoft/azuredatastudio/pull/10918#discussion_r457876752) [](ancestors = 457876752)

Reply:
this is also something I noticed while debugging. I don't know exactly when but while loading the  component. this line of code was hit with no children. so I added the check

---
In reply to: [457876752](https://github.com/microsoft/azuredatastudio/pull/10918#discussion_r457876752) [](ancestors = 457876752)